packages/webcomponents/src/components/checkout/checkout-core.tsx

```diff
@@ -155,6 +155,7 @@ export class CheckoutCore {
             show-card={!this.disableCreditCard}
             show-ach={!this.disableBankAccount}
             show-bnpl={!this.disableBnpl}
+            paymentMethodGroupId={this.paymentMethodGroupId}
```

Comment:
I think we could add the `payment_method_group_id` to the `Checkout` class constructor then we can pass this prop like: 
`paymentMethodGroupId={this.checkout?.payment_method_group_id}`

Reasons:
* Make it consistent with the backend object entity
* Avoid derived state: line 35 and 69

Reply:
Ah this is much better. Thankfully it seems we are already passing this in the Checkout constructor - so its simple enough to just remove lines 35 and 69. 😄